### ISSUE TITLE
Melodic release?

### ISSUE THREAD

**Issue by v-lopez** [orocos/rtt_ros_integration#115]
Would it be possible to release the binaries for ROS melodic?

**Comment by ahoarau** [comment]
With arm64 also ? :)

**Comment by marrasp** [comment]
Is there any news about this?

**Comment by marrasp** [comment]
I mean, about if it would be possible to release the binaries for ROS melodic

**Comment by meyerj** [comment]
Yes, we are currently in the process of drafting a 2.9.1 release with only ABI-compatible patches for ROS kinetic and lunar. Afterwards all the rest will be packed into 2.10 to be released in melodic. Stay tuned, and sorry for the looong delay.

**Comment by marrasp** [comment]
Thank you for the answer

**Comment by Hugal31** [comment]
Any news on this?
